war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2038,7 +2038,7 @@ function updateBuildHistory(ajaxUrl,nBuild) {
         }
 
         function loadPage(params, focusOnSearch) {
-            var searchString = pageSearchInput.value;
+            var searchString = pageSearchInput.value.toLowerCase();
```

Comment:
js kiddy here, but can `pageSearchInput.value` ever be `null` or `undefined` or something that would make this fail?